woocommerce/woocommerce-ios
# Notice audit

This PR updates all of the notices to bring them more inine with the i6 designs. Example:

![3](https://user-images.githubusercontent.com/154014/50614607-ff2cd900-0ea6-11e9-9592-ae9faa7b0a25.png)

In short, I removed all of the titles that appear on the notices.

Fixes #541 

## Testing

1. Review the code/string updates — make sure it makes sense
2. Build and run the app, try some actions that make a notice appear (moderating a review, fulfilling an order, etc)
3. Verify there are no 1-or-2 word titles in the notices — just the message itself.

@mindgraffiti could you take a peek at this for me?
